TelenorNBIoT.cpp

```diff
@@ -475,12 +470,24 @@ int TelenorNBIoT::parseErrorCode(const char *line)
 
 int TelenorNBIoT::splitFields(char *line, char **fields)
 {
-    int found = 1;
     fields[0] = line;
+    int found = 1;
     for(char *p = line; *p != '\0'; p++) {
-        if (*p == ',')
+        if (*p == '"')
+        {
+            // replace double-quote with null-terminator
+            *p = '\0';
```

Comment:
Maybe avoid replacing the opening quote (by doing this in an `else` below).  I can see this causing confusion when debugging down the road.

Reply:
I don't understand what you're suggesting. Please elaborate :)

Reply:
I was confused about where the quotes were – I thought the whole line was quoted, not individual fields.  Ignore me 😛